karate-tests/src/test/java/patients/healthcareWorker/createPatient/TestCreatePatient.java

```diff
@@ -19,11 +19,6 @@ static void beforeAll() {
         }
     }
 
-    @Karate.Test
```

Comment:
not sure of reason for deletion of this - could well be justified, just curious

Reply:
No, it's not important - I sometimes create testSpecific methods just to assist me when testing.

In fact, none of these test runner classes (the junit Java classes) are important when it comes to the CI jobs - they're all just for a bit of convenience when developing and debugging tests. The only ones that matter are the TestParallel.java and TestMockParallel.java files - these are what get called in CI to run all the tests.